protocol/messenger_handler.go

```diff
@@ -1387,8 +1387,19 @@ func (m *Messenger) HandleEditMessage(response *MessengerResponse, editMessage E
 			return err
 		}
 	}
-	response.AddMessage(originalMessage)
-	response.AddChat(chat)
+	responseTo, err := m.persistence.MessageByID(originalMessage.ResponseTo)
+
+	if err != nil && err != common.ErrRecordNotFound {
+		return err
+	}
+
+	originalMessage.JustEdited = true
```

Comment:
I think it would be cleaner if we don't set the field and we have just two different functions:

Something like:

```
updateActivityNotificationCenterMessage(...)
```
and
```
addNew...
```
and then factor out the shared logic inside


or, if you really want to have a single function, then
```
upsertActivityNotification(....., editedMessage bool)
```
instead of modifying message

Reply:
well.. unfortunately I agree

Reply:
@cammellos done, thanks